src/main/java/sg/ncl/MainController.java

```diff
@@ -1103,53 +1080,53 @@ public String editAccountDetails(
             JSONObject userDetails = new JSONObject();
             JSONObject address = new JSONObject();
 
-            userDetails.put("firstName", editUser.getFirstName());
-            userDetails.put("lastName", editUser.getLastName());
-            userDetails.put("email", editUser.getEmail());
-            userDetails.put("phone", editUser.getPhone());
-            userDetails.put("jobTitle", editUser.getJobTitle());
-            userDetails.put("address", address);
-            userDetails.put("institution", editUser.getInstitution());
-            userDetails.put("institutionAbbreviation", originalUser.getInstitutionAbbreviation());
-            userDetails.put("institutionWeb", originalUser.getInstitutionWeb());
-
-            address.put("address1", originalUser.getAddress1());
-            address.put("address2", originalUser.getAddress2());
-            address.put("country", editUser.getCountry());
-            address.put("city", originalUser.getCity());
-            address.put("region", originalUser.getRegion());
-            address.put("zipCode", originalUser.getPostalCode());
-
-            userObject.put("userDetails", userDetails);
+            userDetails.put(FNAME, editUser.getFirstName());
+            userDetails.put(LNAME, editUser.getLastName());
+            userDetails.put(EMAIL, editUser.getEmail());
+            userDetails.put(PHONE, editUser.getPhone());
+            userDetails.put(JOB_TITLE, editUser.getJobTitle());
+            userDetails.put(ADDRESS, address);
+            userDetails.put(INSTITUTION, editUser.getInstitution());
+            userDetails.put(INSTITUTION_ABBREVIATION, originalUser.getInstitutionAbbreviation());
+            userDetails.put(INSTITUTION_WEB, originalUser.getInstitutionWeb());
+
+            address.put(ADDRESS1, originalUser.getAddress1());
+            address.put(ADDRESS2, originalUser.getAddress2());
+            address.put(COUNTRY, editUser.getCountry());
+            address.put(CITY, originalUser.getCity());
+            address.put(REGION, originalUser.getRegion());
+            address.put(ZIP_CODE, originalUser.getPostalCode());
+
+            userObject.put(USER_DETAILS, userDetails);
 
             String userId_uri = properties.getSioUsersUrl() + session.getAttribute(webProperties.getSessionUserId());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)